src/main/java/com/palantir/docker/compose/connection/waiting/MultiServiceHealthCheck.java

```diff
@@ -1,35 +0,0 @@
-/*
- * Copyright 2016 Palantir Technologies, Inc. All rights reserved.
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-
-package com.palantir.docker.compose.connection.waiting;
-
-import static com.google.common.collect.Iterables.getOnlyElement;
-
-import com.google.common.base.Preconditions;
-import com.palantir.docker.compose.connection.Container;
-import java.util.List;
-
-@FunctionalInterface
-public interface MultiServiceHealthCheck {
```

Comment:
Had the conversation on hipchat. Short version is we're going to put this back, this is useful when you can't tell the health of a composition by just looking at services individually.